README.md

```diff
@@ -15,9 +15,7 @@ Google's common Java, C++ and Javascript library for parsing, formatting, storin
   * ` PhoneNumberToCarrierMapper ` - provides carrier information related to a phone number.
 
 # Demo
-[Java Version](http://libphonenumber.appspot.com/) (running code r729 (on Google Code))
-
-[JavaScript Version](http://libphonenumber.googlecode.com/svn/trunk/javascript/i18n/phonenumbers/demo-compiled.html) (running JS code r729 (on Google Code))
```

Comment:
This can be done. Simply searching on Google for "open html file from github" reveals the solution, which is to replace this with:

https://rawgit.com/googlei18n/libphonenumber/master/javascript/i18n/phonenumbers/demo-compiled.html

In case you want more details: http://rawgit.com/